VTEX - Custom Fields API.json

```diff
@@ -0,0 +1,1627 @@
+                            "application/json": {
+                                "schema": {
+                                    "type": "array",
+                                    "description": "Array of custom field value objects matching the search criteria.",
+                                    "items": {
+                                        "type": "object",
+                                        "description": "Custom field value search result object.",
+                                        "properties": {
+                                            "id": {
+                                                "type": "string",
+                                                "description": "The unique ID of the custom field value document."
+                                            },
+                                            "contractId": {
+                                                "type": "string",
+                                                "description": "The ID of the contract associated with the custom field."
+                                            },
+                                            "customFieldId": {
+                                                "type": "string",
+                                                "description": "The ID of the custom field (from `customFieldSettings`)."
+                                            },
+                                            "auxId": {
+                                                "type": "string",
+                                                "description": "Optional auxiliary ID, e.g., address ID for fields with level `address`.",
+                                                "nullable": true
+                                            },
+                                            "value": {
+                                                "type": "string",
+                                                "description": "The value stored in the custom field."
+                                            },
+                                            "description": {
+                                                "type": "string",
+                                                "description": "Optional text describing what the value represents.",
+                                                "nullable": true
+                                            }
+                                        }
+                                    }
+                                },
+                                "examples": {
+                                    "allValues": {
+                                        "summary": "Search all values for a custom field",
+                                        "description": "Returns all values for a specific custom field, sorted by value. Use `_where=contractId={{contractId}} AND customFieldId=\"{{customFieldId}}\"` with `_sort=value ASC`.",
```

Comment:
It's not clear that these are query strings. Please document these as separate query strings. You can find more information in the mdv2 search API docs.